src/components/Dashboard/UserBox.tsx

```diff
@@ -0,0 +1,63 @@
+import { useState } from 'react';
+import classnames from 'classnames';
+import { useRouter } from 'next/router';
+
+import { useUserName } from '../AuthenticationProvider';
+import { useAuthentication } from '../AuthenticationProvider';
+import style from './UserBox.module.scss';
+
+export const UserBox = () => {
+  const [userBoxVisible, setUserBoxVisible] = useState(false);
+
+  const userName = useUserName();
+
+  const router = useRouter();
+
+  const { logout } = useAuthentication();
```

Comment:
I am the biggest fan of new lines. My very subjective feeling is those could be together 

```suggestion
  const userName = useUserName();
  const router = useRouter();
  const { logout } = useAuthentication();
```